lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -2255,6 +2248,7 @@ IRGenDebugInfoImpl::IRGenDebugInfoImpl(const IRGenOptions &Opts,
       }
     OS << '"';
   }
+  createSpecialStlibBuiltinTypes();
```

Comment:
Commit #1 doesn't have any tests, but we should at least be able to test the special types?